python/examples/atomistic/2-running-ase-md.py

```diff
@@ -13,7 +13,7 @@
 .. TODO: once we have more MD engine, add link to their doc here
 """
 
-# sphinx_gallery_thumbnail_number = 4
```

Comment:
why are you changing this if it is commented out?

Reply:
this is the syntax sphinx-gallery uses to have configuration inside the examples: https://sphinx-gallery.github.io/stable/configuration.html#configurations-inside-examples

Reply:
Ahh true I remember now